core/dbt/include/global_project/macros/strategies/materialized_view/create.sql

```diff
@@ -0,0 +1,3 @@
+{% macro strategy__materialized_view__create(relation, sql) %}
```

Comment:
I think we need defaults on these, just like the `db_api` macros. So these would dispatch (`adapter.dispatch(...)`) and then the default would have the body you have here (`db_api__...`). In contrast to the `db_api` macros, I think it's fine to a default strategy that's functional (strategies are probably more reusable and don't directly rely on platform specific sql); so don't raise an exception like we do in the `db_api` macros.

Reply:
added the two defaults